### ISSUE TITLE
[a11y Support] Accredited Representation Management: Find a Representative - Staging Review (mid-Feb 2024)

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#66436]
## Find a Representative - Staging Prep Support

**Definition of done:** CAIA will review the ARM team's "Find a Representative" [staging prototype](https://staging.va.gov/get-help-from-accredited-representative/find-rep) for outstanding accessibility issues by 2/14/24 and document them in the team's [Google Doc](https://docs.google.com/document/d/17sbON_0yuFWUV6g8SjlIfri5uEjSSpMOmwUzVQEJ-RM/edit).

**Staging Review date:** TBD, but planned for mid-February 2024
- [Slack request thread](https://dsva.slack.com/archives/C05L6HSJLHM/p1700865330692189&sa=D&source=calendar&ust=1701526961183839&usg=AOvVaw0Fuy_eW6sLTKZrDiAOVgGL)
- [Collab Cycle ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/65707&sa=D&source=calendar&ust=1701526961183839&usg=AOvVaw3GCH02WwmvjsasKJ5RiaH6)
- [Drafting Template](https://drive.google.com/drive/u/0/folders/1Xr4k7mC0RGOFjbuIU1COqx_gQpQQKVhZ&sa=D&source=calendar&ust=1701526961183839&usg=AOvVaw0HldjWQ21Rqkz2aNimpoNt)

**NEW LINKS NEED ADDED**

[Wireframe received in intake](https://balsamiq.cloud/seq2bvz/po36s5c/r2B02)
[Wireframe in Mural](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1692888741111/2474c0612887653142ab991d234898b6968dbf0e?sender=u1c2b512a71f68e7b5a5c0324)
[DSVA Slack Channel](https://dsva.slack.com/archives/C05L6HSJLHM)
## Auditor(s)
- Jamie Klenetsky Fay @coforma-jamie 

### Product Team
- **OCTO-DE product owner:** Zach Goldfine
- **Product manager:** Lindsay Li-Smith `(submitted the intake ticket)`
- **Designer:** Michael Tri (starting 9/25)
- **Accessibility specialist:** `n/a`
- **Content writer:** n/a
- **Slack channel:** **`benefits-representation-management`**

#### Initial Request
> Our next step is to create wireframes for our MVP. We will be able to use these wireframes to begin outlining a usability test plan and creating a prototype for that usability test, however we do not have a copywriter on our team and will require content assistance from CAIA to complete copy for our prototype.
> **09/25/2023 - our Design Intent is scheduled for this day,** so we hope to have wireframes to review by this date.

#### Review Support
- [ ] Prototype (Midpoint)
- [x] Staging prep artifact

#### Supporting artifacts from CAIA Intake Ticket #65809 
-  [Product outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/accredited-representation-management/product-outline.md)
- [User flows](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1692888741111/2474c0612887653142ab991d234898b6968dbf0e?sender=ud8fb439cf1516edb66c71635)
- [Wireframes](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1692888741111/2474c0612887653142ab991d234898b6968dbf0e?sender=u1c2b512a71f68e7b5a5c0324)
- [Collab Cycle Ticket Opened 9/15/2023](https://app.zenhub.com/workspaces/accredited-representation-management-team-64d0dc51d3e8f4788ac6ef96/issues/gh/department-of-veterans-affairs/va.gov-team/65707)
```[tasklist]
### Tasks
- [x] Create [audit spreadsheet](https://docs.google.com/spreadsheets/d/1-ZkaUf0r4Go6uso-9Nakf2DxOJz4riUmZMTIYihHbK4/edit#gid=0)
- [x] Do audit
- [x] Link to audit in team's Google Doc and share with the team
```

**Comment by sara-amanda** [comment]
@coforma-jamie do we have a recent update on this one post break?

**Comment by coforma-jamie** [comment]
Yes, they ran into some hiccups so staging review won't take place until mid-Feb. I'll be pulled into an a11y review before then.

**Comment by coforma-jamie** [comment]
[Staging prototype](https://staging.va.gov/get-help-from-accredited-representative/find-rep) is ready for review! I'll look it over today and add my findings to their [Google Doc](https://docs.google.com/document/d/17sbON_0yuFWUV6g8SjlIfri5uEjSSpMOmwUzVQEJ-RM/edit), as requested.

**Comment by coforma-jamie** [comment]
Used this [spreadsheet](https://docs.google.com/spreadsheets/d/1-ZkaUf0r4Go6uso-9Nakf2DxOJz4riUmZMTIYihHbK4/edit#gid=0) to track findings. Shared with the team.
